osu.Game.Rulesets.Catch/UI/CatchPlayfield.cs

```diff
@@ -15,7 +15,7 @@ namespace osu.Game.Rulesets.Catch.UI
 {
     public class CatchPlayfield : ScrollingPlayfield
     {
-        public const float BASE_WIDTH = 512;
+        public const float BASE_WIDTH = 640;

```

Comment:
pretty sure this is wrong?

Reply:
When I played osu!catch, some fruits went out of screen!
So I make the playground bigger